cmd/kaspawallet/keys/keys.go

```diff
@@ -157,7 +162,7 @@ func createFileDirectoryIfDoesntExist(path string) error {
 		return nil
 	}
 
-	return os.MkdirAll(dir, 0600)
+	return os.MkdirAll(dir, 0700)
```

Comment:
Why this change?

Reply:
Manually checked and the permissions end up correct:
1. directories need "execution" permissions (to enter).
2. files are still 0600.